src/main.ts

```diff
@@ -29,6 +35,8 @@ async function init() {
 	);
 
 	Engine.Setup(gl);
```

Comment:
Also, since we're 99% using only one webgl context, we might pass the `gl` context in the engine setup step and exporting it from there instead of passing it as a parameter every time :shrug: